latency/src/bin/z_ping.rs

```diff
@@ -72,10 +72,10 @@ async fn single(opt: Opt, config: Config) {
         session
             .put(&reskey_ping, data)
             .congestion_control(CongestionControl::Block)
-            .await
+            .wait()
             .unwrap();
 
-        let mut sample = receiver.next().await.unwrap();
+        let mut sample = receiver.recv().unwrap();
```

Comment:
`recv` and `next` are for sync and async API (used in zn and z files), respectively. Perhaps this is one place that needs to be kept by way of command line arguments.

Reply:
Do you mean that add an option for choosing the usage of `recv` or `next` ?

Reply:
Yes, wr may need to consider adding some options, since z and zn which used differnt features are now put together.